media/CircleGraph/external/circle-schema.js

```diff
@@ -1,8 +1,8 @@
-var $root = flatbuffers.get('circle');
+var cir_root = flatbuffers.get('circle');
```

Comment:
All `$root` is replaced with `cir_root` to avoid conflict with `$root` from `tflite-schema.js`.